types/iframe-resizer/index.d.ts

```diff
@@ -274,4 +274,4 @@ export interface IFrameScrollData {
   y: number;
 }
 
-export function iframeResizer(options: IFrameOptions, target: string | HTMLElement): IFrameComponent[];
+export default function iframeResizer(options: IFrameOptions, target: string | HTMLElement): IFrameComponent[];
```

Comment:
`export default` is almost right here, but not quite. It should be `export = iframeResizer`, followed by `export as namespace iframeResize`. That's based on the source code's commonjs `module.exports = factory()` and global `window.iFrameResize = window.iFrameResize || factory()`. Once you have the commonjs syntax correct, Typescript will emit the correct AMD import code when you import it with `import iframeResizer = require('iframe-resizer')`.